README.md

```diff
@@ -175,7 +175,7 @@ Time interval in seconds for tree view auto refresh, auto refresh has to be enab
 - [Use Azure IoT Hub Toolkit to send and receive messages between your device and IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-vscode-iot-toolkit-cloud-device-messaging)
 - [Use Azure IoT Hub Toolkit for Azure IoT Hub device management](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-management-iot-toolkit)
 - [Quickly build your Azure IoT application with Node.js, Python or REST API](https://devblogs.microsoft.com/iotdev/azure-iot-toolkit-1-2-0-quickly-build-your-azure-iot-application-in-vs-code-with-node-js-python-or-rest-api/)
-- [Azure IoT Hub Toolkit supports C#, Go, Java, Node.js, PHP, Python and Ruby to develop Azure IoT application in VS Code](https://blogs.msdn.microsoft.com/iotdev/2018/08/31/c-go-java-node-js-php-python-or-ruby-choose-your-favorite-language-to-develop-azure-iot-application-in-vs-code/)
+- [Azure IoT Hub Toolkit supports C#, Go, Java, Node.js, PHP, Python and Ruby to develop Azure IoT application in VS Code](https://devblogs.microsoft.com/iotdev/azure-iot-toolkit-supports-c-go-java-node-js-php-python-and-ruby-to-develop-azure-iot-application-in-vs-code/)
```

Comment:
is there any other stale link?

Reply:
All should have redirection. Only this does not work somehow.

Reply:
In readme we still use "Azure IoT Hub Toolkit"?

Reply:
Besides, welcome page seems still use "Azure IoT Hub Toolkit".  Please search the repo and clean.

Reply:
Update "Azure IoT Hub Toolkit" to "Azure IoT Tools" for the docs page. For blog post, we could just keep the original title.

Reply:
How about welcome page?

Reply:
No docs link in welcome page. Page title is already updated.

Reply:
There are still several "Azure IoT Hub Toolkit" in welcome page,  please search the repo and clean. https://github.com/microsoft/vscode-azure-iot-toolkit/blob/master/resources/welcome/index.html#L226

Reply:
Similar to CH9 page: https://github.com/microsoft/vscode-azure-iot-toolkit/blob/e3e6c0265dcb377381e33c3e11eac498e3ac8b04/resources/welcome/index.html#L211, we could just keep the original title as we would not update the original post.

Reply:
It seems confused in the same page with different names for the extension. I would prefer to rename all to "Azure IoT Tools"

Reply:
The title would be better to align with the actual content of the post/docs page. Otherwise, it would be still confusing to customers if they click the link and see the actual content. As it is just a reference link, I think it is OK to keep them with the actual title. We could update if one receive any feedback from end users.

Reply:
Actually, the current "Azure IoT Hub Toolkit" is not same as the titles in Channel9 "Azure Iot Toolkit", since last name change. What we do is make sure in the extension repo we use the new name.